pinot-hadoop/src/main/java/com/linkedin/pinot/hadoop/job/SegmentCreationJob.java

```diff
@@ -84,7 +84,7 @@ public SegmentCreationJob(String jobName, Properties properties) throws Exceptio
     String hostsString = _properties.getProperty(JobConfigConstants.PUSH_TO_HOSTS);
     String portString = _properties.getProperty(JobConfigConstants.PUSH_TO_PORT);
 
-    _defaultPermissions = _properties.getProperty(JobConfigConstants.DEFAULT_PERMISSIONS, null);
+    _defaultPermissionsMask = _properties.getProperty(JobConfigConstants.DEFAULT_PERMISSIONS_MASK, null);
```

Comment:
Lets log the _defaultPermissionsMask.

Reply:
Does the integration test exercise the flow where the mask value is set in _properties from outside?

Reply:
I ran the integration test with/without it set.